src/Fusion.Summary.Api/Controllers/DepartmentsController.cs

```diff
@@ -0,0 +1,100 @@
+﻿using Fusion.Summary.Api.Database.Entities;
+using Fusion.Summary.Api.Services;
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace Fusion.Summary.Api.Controllers;
+
+public record PutDepartmentRquest(string DepartmentSapId, string FullDepartmentName, Guid ResourceOwnerAzureUniqueId);
+
+public record GetDepartmentResponse(string departmentSapId, Guid resourceOwnerAzureUniqueId, string fullDepartmentName);
+
+[Authorize]
+[Route("api/[controller]")]
+[ApiController]
+public class DepartmentsController : ControllerBase
+{
+    private readonly IDepartmentService _departmentService;
+
+    public DepartmentsController(IDepartmentService departmentService) 
```

Comment:
Should try and align design patterns with existing apis perhaps? We have tried to standardise the CQRS pattern. This gives consistency across services for developers and reusability with regards to both concepts and toolings, like audit logging, how to extend with new functionality and debugging..